Bus-Updater/PC_updater_tool/source/src/main/java/org/selfbus/updater/DeviceManagement.java

```diff
@@ -311,14 +315,17 @@ public ResponseResult programBootDescriptor(BootDescriptor bootDescriptor, int d
     public void requestBootLoaderStatistic()
             throws KNXTimeoutException, KNXLinkClosedException, KNXDisconnectException, KNXRemoteException, InterruptedException, UpdaterException {
 
-        byte[] result = sendWithRetry(UPDCommand.REQUEST_STATISTIC, new byte[] {0}, MAX_UPD_COMMAND_RETRY).data();
-        if (result[3] == UPDCommand.RESPONSE_STATISTIC.id)
+        byte[] result = sendWithRetry(UPDCommand.REQUEST_STATISTIC, new byte[] {(byte)0xfe}, MAX_UPD_COMMAND_RETRY).data();
```

Comment:
We should be able to even leave out the dummy byte by passing an empty array: `new byte[0]`

Same applies to calls in `eraseFlash()`, `requestUIDFromDevice()`, and `requestBootDescriptor()`.

Then, these four `UPDCommand`s can be set to `minBytes`=`maxBytes`=`0` in upd_protocol.h.